ngx_http_sxg_filter_module.c

```diff
@@ -37,6 +37,7 @@ typedef struct {
   ngx_str_t cert_url;
   ngx_str_t validity_url;
   ngx_str_t cert_path;
+  size_t expires_seconds;
```

Comment:
Is this really a `size_t` (buffer length)?  Does `unsigned int` more accurately describe it?  nginx has a `ngx_conf_merge_sec_value` helper.

Reply:
OK, `time_t` should be used, `ngx_conf_merge_sec_value` is a helper for `time_t`.